csharp/src/Ice/Reference.cs

```diff
@@ -726,6 +766,7 @@ internal void ClearRequestHandler(IRequestHandler handler)
             Identity? identity = null,
             string? identityAndFacet = null,
             InvocationMode? invocationMode = null,
+            TimeSpan? invocationTimeout = null,
```

Comment:
It doesn't look like we validate the value of invocationTimeout here so if the user pass a zero timeout, it will end up with an assert instead of an ArgumentException like for other arguments with invalid values.

Reply:
Added validation for invocationTimeout in Clone